phpunit.xml.dist

```diff
@@ -1,37 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.5/phpunit.xsd"
          bootstrap="./vendor/autoload.php"
          colors="true"
          failOnIncomplete="true"
          failOnWarning="true"
          failOnRisky="true"
->
-    <coverage>
-        <include>
-            <directory>./src/Sulu/Component/</directory>
-        </include>
-
-        <exclude>
-            <directory>./tests</directory>
-            <directory>./vendor</directory>
-            <directory>./*/Tests</directory>
-        </exclude>
-    </coverage>
-
+         cacheDirectory=".phpunit.cache">
     <testsuites>
         <testsuite name="components">
             <directory suffix="Test.php">./src/Sulu/Component/*/Tests</directory>
         </testsuite>
     </testsuites>
-
     <php>
         <server name="APP_ENV" value="test" force="true"/>
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak"/>
         <ini name="date.timezone" value="UTC"/>
     </php>
-
-    <listeners>
-        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener"/>
-    </listeners>
+    <source>
```

Comment:
if we move this up to the old position of `<coverage>` it would not mess up the git diff for include and exclude.

Reply:
Good idea. Did that.